ReText/window.py

```diff
@@ -943,8 +944,40 @@ def standardPrinter(self, title):
 		printer = QPrinter(QPrinter.HighResolution)
 		printer.setDocName(title)
 		printer.setCreator('ReText %s' % app_version)
+		if globalSettings.paperSize:
+			pageSize = self.getPageSizeByName(globalSettings.paperSize)
+			if pageSize is not None:
+				printer.setPaperSize(pageSize)
+			else:
+				QMessageBox.warning(self, '',
+					self.tr('Unrecognized PaperSize setting "%s".' %
```

Comment:
The closing `)` is in wrong place, this won’t work. The translators need to translate the string with literal `%s`, not with the substituted value.

It should be `self.tr('Unrecognized PaperSize setting "%s".') % globalSettings.paperSize`.

Reply:
Done.